### PR TITLE
Fix MaaS_ML and MaaS_DL exec commands

### PR DESCRIPTION
**Pull request by ImenBizid**

### CODE REVIEW

File: MaaSMLPCA/resources/catalog/MaaS_ML.xml [hunk 1]
```diff
@@ -479,9 +479,13 @@ else
...
     echo "docker exec $INSTANCE_NAME ls -la $INSTANCE_PATH"
     docker exec $INSTANCE_NAME ls -la $INSTANCE_PATH
-    echo "docker exec -dit $INSTANCE_NAME bash -c python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
-    docker exec -dit $INSTANCE_NAME  bash -c "python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
-    
+    if [ "${GPU_ENABLED,,}" = "true" ]; then
+        echo "docker exec -dit $INSTANCE_NAME bash -c /opt/conda/envs/rapids/bin/python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
+        docker exec -dit $INSTANCE_NAME  bash -c "/opt/conda/envs/rapids/bin/python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
+    else
+    	echo "docker exec -dit $INSTANCE_NAME bash -c python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
```
Comment (dianajlailaty) [inline MaaSMLPCA/resources/catalog/MaaS_ML.xml:486]: I think there should be an additional space before echo

**Review by dianajlailaty — CHANGES_REQUESTED** [review]

**Review by dianajlailaty — APPROVED** [review]

File: MaaSDLPCA/resources/catalog/MaaS_DL.xml [hunk 2]
```diff
@@ -470,7 +470,7 @@ else
     echo "docker exec $INSTANCE_NAME ls -la $INSTANCE_PATH"
     docker exec $INSTANCE_NAME ls -la $INSTANCE_PATH
     echo "docker exec -dit $INSTANCE_NAME bash -c python $INSTANCE_PATH/dl_service.py >> /proc/1/fd/1"
-    docker exec -dit $INSTANCE_NAME bash -c "python $INSTANCE_PATH/dl_service.py"
+    docker exec -dit $INSTANCE_NAME bash -c "python $INSTANCE_PATH/dl_service.py  >> /proc/1/fd/1"
```
Comment (mboussaa) [inline MaaSDLPCA/resources/catalog/MaaS_DL.xml:473]: why detached and interactive at the same time?

File: MaaSMLPCA/resources/catalog/MaaS_ML.xml [hunk 3]
```diff
@@ -479,9 +479,13 @@ else
     docker cp certificate_mas.pem $INSTANCE_NAME:$INSTANCE_PATH/certificate_mas.pem
     echo "docker exec $INSTANCE_NAME ls -la $INSTANCE_PATH"
     docker exec $INSTANCE_NAME ls -la $INSTANCE_PATH
-    echo "docker exec -dit $INSTANCE_NAME bash -c python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
-    docker exec -dit $INSTANCE_NAME  bash -c "python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
-    
+    if [ "${GPU_ENABLED,,}" = "true" ]; then
+        echo "docker exec -dit $INSTANCE_NAME bash -c /opt/conda/envs/rapids/bin/python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
+        docker exec -dit $INSTANCE_NAME  bash -c "/opt/conda/envs/rapids/bin/python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
```
Comment (mboussaa) [inline MaaSMLPCA/resources/catalog/MaaS_ML.xml:484]: same. It should be either one

File: MaaSMLPCA/resources/catalog/MaaS_ML.xml [hunk 4]
```diff
@@ -479,9 +479,13 @@ else
...
+    else
+        echo "docker exec -dit $INSTANCE_NAME bash -c python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
+        docker exec -dit $INSTANCE_NAME  bash -c "python $INSTANCE_PATH/ml_service.py >> /proc/1/fd/1"
```
Comment (mboussaa) [inline MaaSMLPCA/resources/catalog/MaaS_ML.xml:487]: same

File: MaaSDLPCA/resources/catalog/MaaS_DL.xml (hunk 2 above)
Comment (ImenBizid) [reply MaaSDLPCA/resources/catalog/MaaS_DL.xml:473]: yes you are right, I will remove it

**Review by andrewssobral — APPROVED** [review]

File: MaaSDLPCA/resources/catalog/MaaS_DL.xml (hunk 2 above)
Comment (mboussaa) [reply MaaSDLPCA/resources/catalog/MaaS_DL.xml]: @ImenBizid too late, @andrewssobral merged :/

Comment (andrewssobral) [reply MaaSDLPCA/resources/catalog/MaaS_DL.xml]: @mboussaa not late, @ImenBizid fixed the issue before the merge.

Comment (mboussaa) [reply MaaSDLPCA/resources/catalog/MaaS_DL.xml]: Ok thx :))

**Pull request merged**